src/aro/Parser.zig

```diff
@@ -7680,6 +7680,8 @@ fn primaryExpr(p: *Parser) Error!Result {
                 else
                     try p.errStr(.implicit_func_decl, name_tok, name);
 
+                if (p.record.kind != .invalid) return .{ .ty = Type.invalid };
```

Comment:
This line looks quite strange out of context, could you add a small comment explaining it? Something like `do not create implicit functions when parsing record declarators` if I'm understanding correctly.